### ISSUE TITLE
epics.put works but epics.get returns wrong number

### ISSUE THREAD

**Issue by bsobhani** [epics-modules/script#12]
I am trying to read from and write to an ao record called testRecord2. The IOC for testRecord is running completely independently of the lua module, in an IOC called "example".

First I run the example lua shell by doing "../../bin/linux-x86_64/testLuaShell st.lua" in the "lua/iocBoot/iocLuaShell" directory. epics.get("testRecord2") for some reason always returns 1.581010066692e-322. But when I do epics.put("testRecord2", 3), it sets the record to the right number (3), which I can verify by doing "caget testRecord2" in bash. But when I do epics.get("testRecord2") in the luash shell, it continues to return 1.581010066692e-322.

I am running the latest version of epics 7.0, compiled from the github repo, on Debian 9.5. I have tried both the latest asyn from the github repo, as well as asyn 4-33, but they both result in the same behavior.

"cainfo testRecord" returns:

testRecord2
    State:            connected
    Host:             ...
    Access:           read, write
    Native data type: DBF_DOUBLE
    Request type:     DBR_DOUBLE
    Element count:    1

**Comment by keenanlang** [comment]
Issue was due to a dumb copy-paste error. I wrote the epics.get code with the same structure as epics.put, however that meant that the command to handle channel access events happened *after* I had tried to grab the value. Every time I had used it since I wrote it, I was using PV's on the same IOC, so it wasn't using channel access and I wasn't noticing.

Things should be working correctly with the latest commit (38cea60), I also added an optional timeout parameter to the command.
